pybliometrics/utils/get_content.py

```diff
@@ -67,30 +67,43 @@ def get_content(url, api, params=None, **kwds):
     from random import shuffle
     from time import sleep, time
 
+    # Get needed ressources for query
     config = get_config()
+
     keys = get_keys()
+
+    # Get tokens and zip with keys
+    insttokens = get_insttokens()
+    insttokens = list(zip(keys, insttokens))
+
+    # Keep keys that are not insttokens
+    keys = keys[len(insttokens):]
+
     session = get_session()
 
-    # Set header, params and proxy
-    try:
-        header = {'X-ELS-APIKey': keys[0],
-                  'Accept': 'application/json',
-                  'User-Agent': user_agent}
-    except IndexError:
-        raise errors[429]
-
-    if config.has_option('Authentication', 'InstToken'):
-        token = config.get('Authentication', 'InstToken')
-        header['X-ELS-Insttoken'] = token
     params = params or {}
     params.update(**kwds)
     proxies = dict(config._sections.get("Proxy", {}))
+    timeout = config.getint("Requests", "Timeout", fallback=20)
 
-    # Replace credentials if provided
-    if "apikey" in params:
-        header['X-ELS-APIKey'] = params.pop("apikey")
+    # Base header
+    base_header = {'Accept': 'application/json',
+                   'User-Agent': user_agent}
```

Comment:
if we move this further down, we can add the key, which always needs to be present.